### PR TITLE
Add data collection practices document

### PR DESCRIPTION
**Pull request by colinxfleming**
We had a conversation started by @lwaldsc  in slack stemming from another org asking for data collection best practices about whether it might be worth making a reference dump on how we've approached these problems for other groups doing social justice work using sensitive data.

Here's what I wrote on the plane before falling asleep. Obviously a work in progress. People should feel free to push straight to this branch to add their thoughts (after I flesh it out a little more and complete, like, sentences).

This pull request makes the following changes:
* Add doc about our data collection practices

No issues.

cc @lwaldsc @cfinucane @lomky @DarthHater , which I think is the core group of people who have been thinking about this a lot. Others should feel free to chime in!

### CODE REVIEW

**Comment by lwaldsc** [comment]
Tagging issue `#812 `

**Comment by colinxfleming** [comment]
Turns out writing is a lot easier when you aren't way underslept! I've got what I believe to be a working draft of this that I'd like other people's opinions and eyes on.

**Comment by DarthHater** [comment]
I'll take a gander once I'm back from John Denvertopia

File: docs/OUR_DATA_COLLECTION_PRACTICES.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+### An example of something borderline that we store but don't export: Special circumstances
+
+We store flags related to a patient's circumstances, so that if a patient is dealing with a complex situation our case managers can direct them to helpful resources.
+
+However, in the unlikely event that an anonymized export were to leak, we wouldn't want potentially harmful information or intimate details about a patient to leak along with them. So while we store that information, it's accessible only through the case manager workflow and not through the exports.
+
+## How long we store it
+
+We store anonymized reporting data indefinitely; personally identifiable fields are wiped after (TBD EXACT BUSINESS RULES).
```
Comment (lomky) [inline docs/OUR_DATA_COLLECTION_PRACTICES.md]: Maybe throw a link to the archive issue here, so anyone curious can see the TBD in process?

Comment (lwaldsc) [inline docs/OUR_DATA_COLLECTION_PRACTICES.md]: Seconding @lomky on throwing #812 in here.

Comment (colinxfleming) [inline docs/OUR_DATA_COLLECTION_PRACTICES.md]: A+ flag. Racked in df1b939

**Comment by colinxfleming** [comment]
Thanks much @lomky and @lwaldsc for your eyes!

Let's let this hang just in case for Dee @ DCAF to take a look at this coming week, and plan on closing it mid next week either way. I'd rather have it in and pull requestable than hanging out in the ether like it is now.

**Comment by lwaldsc** [comment]
💰💰💰💰💰💰💰

**Comment by colinxfleming** [comment]
attn @deesriv -- I'm gonna merge this but please flag any thoughts you have for @lwaldsc and @NerdyGirl537

**Pull request merged**